tests/test_volcano_plot.py

```diff
@@ -1,96 +1,17 @@
 import os
-from selenium.webdriver.common.keys import Keys
-from pytest_dash.wait_for import (
-    wait_for_text_to_equal,
-    wait_for_element_by_css_selector,
-    wait_for_elements_by_css_selector
-)
+
 from dash_bio import VolcanoPlot
 from dash_bio.component_factory._volcano import GENOMEWIDE_LINE_LABEL, \
     EFFECT_SIZE_LINE_MIN_LABEL, EFFECT_SIZE_LINE_MAX_LABEL
 from tests.dashbio_demos.app_volcano_plot import DATASETS
 from .test_common_features import (
-    init_demo_app,
     template_test_python_component_prop,
     PROP_TYPES
 )
 
 APP_NAME = os.path.basename(__file__).replace('test_', '').replace('.py', '').replace('_', '-')
 
 
-# Demo app tests
-@init_demo_app(APP_NAME)
-def test_click_app_link_from_gallery(dash_threaded):
```

Comment:
Actually, we currently have this test for other demo apps. Are you sure you want to take it out?

Reply:
Whoops! Good catch :)